src/actions.js

```diff
@@ -0,0 +1,18 @@
+export const ADD_ITEM = 'ADD_ITEM'
+
+export const addShirt = () => ({
```

Comment:
instead of having 4 action creators, you could have had one action creator that accept an argument.